css/main.css

```diff
@@ -133,36 +133,52 @@ a:hover {
   padding: 0 10px;
 }
 
-.listings {
+#listings {
```

Comment:
why the change from class to ID?

Reply:
I changed the listings element from a class to an id because it's part of the layout, so we'll only have one listings element which we populate with many entry class elements.